Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -235,6 +236,12 @@ public static class MacroHelper
             { "%pg2", PlayerPronoun2 }, // Him/Her (player)
             { "%pg2self", PlayerPronoun2self },// Himself/Herself (player)
             { "%pg3", PlayerPronoun3 },  // His/Her (player)
```

Comment:
Do we need capitalized versions of `%pgXX` macros? 
Like in
```
<ce> "He ran this way!"
<ce> You started hearing
<ce> the sound of heavy boots behind you.
```

Reply:
This PR's intended scope is to resolve issues with classic quests (I found up to 80 that have this issue). If someone wants to make new quests that use uncapitalized macros at the start of a sentence, those macros can be added as needed; otherwise, we'd be making new macro handlers for every macro that can return an uncapitalized string.

Reply:
I wouldn't say every. Player pronouns to match character pronouns makes sense from a consistency point of view. I still thinks the PR makes sense as it is